web/privacy.html

```diff
@@ -22,64 +25,157 @@
-          di legge, nonché opporsi in ogni caso, per motivi legittimi, al loro trattamento. Le richieste vanno inviate al
-          seguente indirizzo: <a href="mailto:info@agid.gov.it">info@agid.gov.it</a>. 
-          Attraverso il medesimo indirizzo, l’Utente potrà chiedere inoltre la lista
-          aggiornata di tutti i Responsabili del trattamento nominati dal Titolare.
-        </p>
-        <p>
-          La presente Privacy Policy è soggetta ad aggiornamenti; gli Utenti sono pertanto invitati a verificarne periodicamente il
-          contenuto.
-        </p>
-        <p>
-          <h2>Cookies policy</h2>
-          <p>
-            I cookies sono piccoli file di testo che i siti visitati inviano al terminale dell’Utente, dove vengono memorizzati, per
-            poi essere ritrasmessi agli stessi siti alla visita successiva. L’eventuale utilizzo di Cookie - o di altri strumenti
-            di tracciamento - da parte di questa Applicazione o dei titolari dei servizi terzi utilizzati da questa Applicazione,
-            ove non diversamente precisato, ha la finalità di fornire il servizio richiesto dall'Utente. I cookies utilizzati
-            dal Sito non consentono la raccolta di informazioni personali relative all’utente e si cancellano automaticamente
-            con la chiusura del browser di navigazione. I cookies non sono utilizzati per attività di profilazione dell’Utente.
-            L’Utente può scegliere di abilitare o disabilitare i cookies intervenendo sulle impostazioni del proprio browser
-            di navigazione secondo le istruzioni rese disponibili dai relativi fornitori.
-          </p>
+        La presente informativa descrive le modalità del trattamento dei dati personali degli utenti (di seguito gli “Utenti”) che consultano il portale raggiungibile all’indirizzo <a href="https://developer.io.italia.it">developer.io.italia.it</a> (di seguito il "Portale").
+        </p>
+        <p>
+        L’informativa è resa ai sensi dell’art. 13 del <a ref="https://eur-lex.europa.eu/legal-content/IT/TXT/HTML/?uri=CELEX:32016R0679#d1e2261-1-1">Regolamento (UE) 2016/679</a> (di seguito il “Regolamento”). La validità dell’informativa contenuta nella presente pagina è limitata al solo Portale e non si estende ad altri siti web eventualmente consultabili mediante collegamento ipertestuale.
+        </p>
+        <br>
+        <p>
+        <h3>Data protection officer (DPO) o Responsabile Protezione Dati (RPD)</h3>
+        La società PagoPA Spa ha nominato un proprio Responsabile della Protezione dei dati, ai sensi dell’art. 37 del Regolamento UE 2016/679, che può essere contattato ai seguenti recapiti:
+        <ul>
+            <li>mail: <a href="mailto:dpo@pagopa.it">dpo@pagopa.it</a>;</li>
+        <li>PEC: <a href="mailto:dpo@pec.pagopa.it">dpo@pec.pagopa.it</a>;</li>
+        <li>indirizzo: Via Sardegna n. 38 - 00187, ROMA (sede operativa della società)</li>
+</ul>
```

Comment:
Please check document indentation

Reply:
Tag position (which is messy on this case) does not affect formatting, so it is more a beauty change. I will tide up the code anyway to make it more readable for review and future changes. Thanks!